pkg/arm/arm.go

```diff
@@ -50,8 +50,6 @@ func (g *simpleGenerator) Generate(ctx context.Context, cs *api.OpenShiftManaged
 			vnet(cs),
 			ipAPIServer(cs),
 			lbAPIServer(&g.pluginConfig, cs),
-			storageRegistry(cs),
```

Comment:
can we leave storageRegistry in the ARM template?

Reply:
ok, sure